### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by varun-s22**
## New UI

Screenshot 2023-12-18 at 00 45 01
Screenshot 2023-12-18 at 00 45 08

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -12,6 +12,37 @@
...
+
+      // Time calculations for hours, minutes and seconds
+      const hours = Math.floor(
+        (distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)
+      );
+      const minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+      const seconds = Math.floor((distance % (1000 * 60)) / 1000);
+
+      // Display the result in the element with id="demo"
+      document.querySelector(".header-timer-space").innerHTML =
```
Comment (sohomdatta1) [inline templates/index.html:37]: Let's make this innerText

**Comment by gopavasanth** [comment]
The description doesn't seem very appealing to me, what do you think? I feel it is self explanatory.

**Comment by varun-s22** [comment]
I had a chat with Sudhandhu and he suggested that we should have a short description about the tool

File: templates/index.html (hunk 1 above)
Comment (varun-s22) [reply templates/index.html:37]: sure

**Review by gopavasanth — APPROVED** [review]

**Pull request merged**